core/platform/models.py

```diff
@@ -58,8 +58,8 @@ class Platform(python_utils.OBJECT):
     """A base class for platform-specific imports related to GAE."""
 
     @classmethod
-    def import_models(cls):
-        # type: () -> None
+    def import_models(cls, unused_model_names=['']):
```

Comment:
Don't include a default arg here. It's not needed and '' doesn't make sense semantically anyway in this context (it's not a reasonable default arg).

Reply:
Removed!